task/testmap.py

```diff
@@ -102,6 +102,10 @@
             'rhel-7-7/openstack',
             'rhel-7-7/vmware',
         ],
+        # These can be triggered manually with bots/tests-trigger
```

Comment:
It is now only `tests-trigger` not `bots/tests-trigger` :) but that is the smallest nitpick and also it is used in the file on other places as well, so not something that need repush :)

Reply:
Oops, sorry for that, I missed this as I just copied the already existing lines together with the comment. As for your question -- yes, the goal is to enable manual triggering of `rhel-7-7/tar` in `weldr/lorax` repository`. Thank you :)